Header header
float64 angle
float64 elapsed_time
int64 current_trial_index
TrialDataMagnoArena trial_data


